zebra-chain/src/parameters/network.rs

```diff
@@ -125,9 +226,30 @@ impl Network {
         self.to_string().to_ascii_lowercase()
     }
 
+    /// Return the network cache name.
+    pub fn cache_name(&self) -> String {
+        self.testnet_parameters()
+            .cache_name()
+            .map_or_else(|| self.lowercase_name(), ToString::to_string)
+    }
+
+    /// Returns `true` if this network is `Mainnet`.
+    pub fn is_mainnet(&self) -> bool {
+        *self == Network::Mainnet
+    }
+
     /// Returns `true` if this network is a testing network.
-    pub fn is_a_test_network(&self) -> bool {
-        *self != Network::Mainnet
+    pub fn is_testnet(&self) -> bool {
```

Comment:
The method name `is_a_test_network()` was chosen specifically so that reviewers could see that it was `true` for any testnet, not just the default public testnet. So let's not change it?